DEVELOPMENT.md

```diff
@@ -0,0 +1,62 @@
+# Welcome
+If you would like the latest stable release of the extension, please reference the README.md for instructions on how to retrieve that.
+You can download a zip archive of the source code for the extension. You can extract this to any directory of your choice and open it within vscode to begin poking around.
+
+The project currently has many components and is growing, please refer to the wiki for an overview of what the extension includes. There is also user documentation on the right side with additional release specific use case guides. Here is the one for v1.3.1:
+https://github.com/apache/daffodil-vscode/wiki/Apache-Daffodil%E2%84%A2-Extension-for-Visual-Studio-Code:-v1.3.1 
+
+## Contributing
+Due to this being an Apache project, if you would like to contribute, you will need to fork the daffodil-vscode main branch to your own repo of choice and create pull requests to the main branch with your code changes.
+ ![image](https://github.com/ctc-oss/daffodil-vscode/assets/131286323/160ff687-4b37-4730-b867-128d063950b7)
+ 
+Once forked, you can clone that forked repository to your own local environment. This can be done by using `git clone`, followed by copy and pasting the https or ssh urls shown under the “Code” dropdown.
+ ![image](https://github.com/ctc-oss/daffodil-vscode/assets/131286323/e32b5ebf-a45d-4362-aa44-5aa817d4bd0b)
+
+ ![image](https://github.com/ctc-oss/daffodil-vscode/assets/131286323/3e123e64-70bb-4f5d-87c9-08f28428630c)
```

Comment:
Recommend changing this `git clone` image to one with a generic (i.e., fake) username